meteor/lib/collections/ExpectedPackages.ts

```diff
@@ -0,0 +1,97 @@
+import { ExpectedPackage } from '@sofie-automation/blueprints-integration'
+import { TransformedCollection } from '../typings/meteor'
+import { registerCollection, Time, ProtectedString } from '../lib'
+import { createMongoCollection } from './lib'
+import { RundownId } from './Rundowns'
+import { ResultingMappingRoutes, StudioId } from './Studios'
+import { PieceId } from './Pieces'
+import { registerIndex } from '../database'
+import { AdLibActionId } from './AdLibActions'
+
+export type ExpectedPackageId = ProtectedString<'ExpectedPackageId'>
+
+export type ExpectedPackageDB =
+	| ExpectedPackageDBFromPiece
+	// | ExpectedPackageDBFromAdlibPiece
+	| ExpectedPackageDBFromAdLibAction
+
+export enum ExpectedPackageDBType {
+	PIECE = 'piece',
+	// ADLIB_PIECE = 'adlib_piece',
+	ADLIB_ACTION = 'adlib_action',
+}
+export interface ExpectedPackageDBBase extends ExpectedPackage.Base {
+	_id: ExpectedPackageId
+
+	/** The studio of the Rundown of the Piece this package belongs to */
+	studioId: StudioId
+	/** The rundown of the Piece this package belongs to */
+	rundownId: RundownId
```

Comment:
What about Bucket AdLibs? I think this needs to be either rundownId or bucket ad lib id.

Reply:
Let's bring that up on the workshop on Friday, I think we'll have a lot more stuff that'll need ironing out 🙂

Reply:
Something else to consider is that we should probably allow both Pieces and PieceInstances to create ExpectedPackages. Otherwise an ingest change could free/delete a resource/asset that is currently on air

Reply:
I've implemented support for the bucket adlibs now.
I added _Pieces and PieceInstances to create ExpectedPackages_ to the todo-list